packages/core/core/src/requests/PathRequest.js

```diff
@@ -168,12 +168,15 @@ export class ResolverRunner {
         }
       }
     } else {
-      if (
-        dep.specifierType === 'url' &&
-        dependency.specifier.startsWith('//')
-      ) {
-        // A protocol-relative URL, e.g `url('//example.com/foo.png')`
-        return null;
+      if (dep.specifierType === 'url') {
```

Comment:
Ideally we would let plugins that wanted to handle urls do so. Maybe this should be in the default resolver plugin instead?

Reply:
I agree it would make sense to have a check like this in something like the default resolver plugin. However I can't see another way of bypassing the `if (typeof parsed.pathname !== 'string')` block around line 184/187 below which throws an error when it attempts to parse a "path" like `#clip-path`.

I tried handling the case in `packages/resolvers/default/src/DefaultResolver.js` but nothing seems to work, I suspect because the error is being thrown before any resolver plugins get hit. Part of the reason I chose to modify the code in `PathRequest` is because it saw it handling a similar case for paths starting with `//`.

I'm not that familiar with the structure of the Parcel codebase, so if you have any suggestions for a better way to handle the case of a path starting with `#` I'm happy to try it.

Reply:
Ok, thanks. Given that this is already the case, this is fine for now.